core/qthelper.cpp

```diff
@@ -443,6 +443,16 @@ QString getUiLanguage()
 	return prefs.locale.lang_locale;
 }
 
+// the water types need to match the watertypes enum
+// since the translations aren't loaded until after the app starts, we can't simply initialize
+// that variable as a const, we need to have an init function that does this at runtime.
+QStringList waterTypes;
+
+void setupTranslatedStrings() {
+	waterTypes = QStringList() << gettextFromC::tr("Fresh") << gettextFromC::tr("Brackish") << gettextFromC::tr("EN13319") << gettextFromC::tr("Salt") << gettextFromC::tr("Use DC");
```

Comment:
BTW: I never understood that pattern. Seems a) less readable and b) gives less chance of optimization when compared to
```
    waterTypes = QStringList {
        gettextFromC::tr("Fresh"), gettextFromC::tr("Brackish"), ...
    };
```
In the latter case, the array can be initialized with the correct size from the start.